tests/back/Pim/Enrichment/Integration/Storage/ElasticsearchAndSql/ProductAndProductModel/WithRemovedAttributeTestCase.php

```diff
@@ -0,0 +1,321 @@
+<?php
+
+namespace AkeneoTest\Pim\Enrichment\Integration\Storage\ElasticsearchAndSql\ProductAndProductModel;
+
+use Akeneo\Pim\Enrichment\Component\Product\Model\ProductInterface;
+use Akeneo\Pim\Enrichment\Component\Product\Model\ProductModelInterface;
+use Akeneo\Pim\Structure\Component\Model\AttributeInterface;
+use Akeneo\Pim\Structure\Component\Model\FamilyInterface;
+use Akeneo\Pim\Structure\Component\Model\FamilyVariantInterface;
+use Akeneo\Test\Integration\Configuration;
+use Akeneo\Test\Integration\TestCase;
+use PHPUnit\Framework\Assert;
+
+abstract class WithRemovedAttributeTestCase extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->loadFixtures();
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function getConfiguration(): Configuration
+    {
+        return $this->catalog->useMinimalCatalog();
+    }
+
+    protected function loadFixtures(): void
+    {
+        $this->createAttribute([
```

Comment:
```suggestion
        $this->createAttributes([code_1, code_2, etc]);
```

As it's always text and `other` group, a simple foreach would do the stuff and it would be more readable.

Reply:
It is a good advice but I need to create `boolean` attributes for variation axes here